File: FrameIncam.Client.FreeLancer/ng/domains/master/customer/masterCustomer.js
(function ($, ng, debugMode, undefined) {
    "use strict";

    // --------------------------------------------------------------------------------
    // Dependencies
    // --------------------------------------------------------------------------------
    var module = ng.module("app");

    // --------------------------------------------------------------------------------
    // Model
    // --------------------------------------------------------------------------------
    module.run([
        "services.utils",
        "services.models",
        function (
            p_utils,
            p_models
        ) {
            // --------------------------------------------------------------------------------
            // Namespace
            // --------------------------------------------------------------------------------
            var fincam = p_models.namespace("fincam");
            var customer = p_models.namespace("fincam.master.customer");

            // --------------------------------------------------------------------------------
            // Model
            // --------------------------------------------------------------------------------
            p_models.class({
                namespace: customer,
                name: "masterCustomer",
                base: {
                    namespace: fincam,
                    name: "logModel"
                },
                properties: {
                    id: "int",
                    code: "int",
                    name: "string",
                    email: "string",
                    mobile: "string",
                    alternateMobile: "string",
                    pincode: "string",
                    addressLine1: "string",
                    addressLine2: "string",
                    cityGeoId: "int",
                    city: "string",
                    stateGeoId: "int",
                    state: "string",
                    isactive: "int",
                    password:"string"
                },
                prototype: new function () {
                    var self = this;
                    return self;
                }
            });
        }
    ]);

})(jQuery, angular, document.children[0].hasAttribute("debug"));

File: FrameIncam.Domains/Models/Master/Customer/MasterCustomer.cs
using MySqlX.XDevAPI.Relational;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace FrameIncam.Domains.Models.Master.Customer
{
    [Table("master_customer")]
    public class MasterCustomer : LogModel
    {
        [Column("code")]
        public int? Code { get; set; }
        [Column("name")]
        public string Name { get; set; }
        [Column("email")]
        public string Email { get; set; }
        [Column("mobile")]
        public string Mobile { get; set; }
        [Column("alternate_mobile")]
        public string AlternateMobile { get; set; }
        [Column("pincode")]
        public string Pincode { get; set; }
        [Column("address_line_1")]
        public string AddressLine1 { get; set; }
        [Column("address_line_2")]
        public string AddressLine2 { get; set; }
        [Column("city_geo_id")]
        public int? CityGeoId { get; set; }
        [Column("city")]
        public string City { get; set; }
        [Column("state_geo_id")]
        public int? StateGeoId { get; set; }
        [Column("state")]
        public string State { get; set; }
        [Column("is_active")]
        public Int16 IsActive { get; set; } = 1;
        [NotMapped]
        public string Password { get; set; }
        public bool IsValid()
        {
            return !string.IsNullOrEmpty(Email) && !string.IsNullOrEmpty(Name) && !string.IsNullOrEmpty(Mobile);
        }
    }
}
